runtime/gc_modron_startup/mmparseXgc.cpp

```diff
@@ -254,6 +254,17 @@ j9gc_initialize_parse_gc_colon(J9JavaVM *javaVM, char **scan_start)
 		}
 		goto _exit;
 	}
+
+	if(try_scan(scan_start, "concurrentScavengeAllocDeviationBoost=")) {
```

Comment:
This seems unrelated to DDR: why is this here?

Reply:
Oops, accidentally pulled in an unrelated change